css/search.css

```diff
@@ -1,15 +1,13 @@
-.jobs-unified-top-card__job-insight--highlight,
-div.jobs-unified-top-card__primary-description-without-tagline
-  > div
-  > span:last-child,
+/* .jobs-unified-top-card__job-insight--highlight,
+div.jobs-unified-top-card__primary-description-without-tagline:last-child,
 li.job-card-container__footer-item > strong,
 li.job-card-container__applicant-count,
```

Comment:
These changes were hard to read so moved some of the code to separate lines with comments.